src/main.rs

```diff
@@ -108,6 +109,40 @@ fn parse_matches() -> clap::ArgMatches {
                 .takes_value(true)
                 .help("Genesis config. bootstrap validator stake sol"),
         )
+        //Docker config
+        .arg(
+            Arg::with_name("skip_docker_build")
+                .long("skip-docker-build")
+                .help("Skips build Docker images"),
+        )
+        .arg(
+            Arg::with_name("registry_name")
+                .long("registry")
+                .takes_value(true)
+                .required(true)
```

Comment:
is this required if `skip-docker-build` is set? I wonder if this and all the other new flags should conflict with `skip-docker-build`

Reply:
yes it is because when we go do pull docker images for deploying kubernetes pods, we need to know what registry, image name, etc to pull.

Reply:
ah gotcha, thanks!